src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -219,11 +219,11 @@ export default {
         },
         addConditionSet() {
             let conditionSetDomainObject;
-            const handleItemSelection = (item) => {
+            function handleItemSelection(item) {
                 if (item) {
                     conditionSetDomainObject = item;
                 }
-            };
+            }
             const dismissDialog = (overlay, initialize) => {
```

Comment:
This function should also violate `func-style`. Why is it not getting flagged?

Reply:
See comment above in StylesView.vue